### PR TITLE
Added uptime conversions

### PR DESCRIPTION
**Pull request by qduk**

### CODE REVIEW

File: tests/unit/test_utils.py [hunk 1]
```diff
@@ -18,11 +23,49 @@
...
+        "received": "2 hours, 50 seconds",
+    },
+    {
+        "sent": 698490,
+        "received": "1 week, 1 day, 2 hours, 1 minute, 30 seconds",
+    },
+]
+
+UPTIME_TO_SECONDS = [
+    {"sent": "1 week, 0 days, 0 hours, 10 minutes", "received": 605400},
```
Comment (itdependsnetworks) [inline tests/unit/test_utils.py:60]: Make sure to have years included in the tests as well.

**Review by jeffkala — APPROVED** [review]

File: tests/unit/test_utils.py (hunk 1 above)
Comment (qduk) [reply tests/unit/test_utils.py:64]: Updated

File: README.md [hunk 2]
```diff
@@ -30,6 +32,7 @@ Functions are grouped with like functions, such as IP or MAC address based funct
 * Ping - Provides the ability to ping, currently only tcp ping.
 * Protocol Mapper - Provides a mapping for protocol names to numbers and vice versa.
 * Route - Provides the ability to provide a list of routes and an IP Address and return the longest prefix matched route.
+* Time -Provides the ability to convert between integer time and string times.
```
Comment (jeffkala) [inline README.md:35]:
```suggestion
* Time - Provides the ability to convert between integer time and string times.
```

File: netutils/time.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
+"""Functions for working with time."""
+import re
+from .constants import TIME_MAPPINGS, UPTIME_REGEX_PATTERNS
+
+
+def uptime_seconds_to_string(uptime_seconds):
+    """Converts uptime in seconds to uptime in string format.
```
Comment (jeffkala) [inline netutils/time.py:7]:
```suggestion
    """Converts uptime in seconds to uptime in human readable string format.
```

File: netutils/time.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            uptime_seconds -= value * count
+            if value == 1:
+                interval = interval.rstrip("s")
+            result.append(f"{value} {interval}")
+
+    return ", ".join(result)
+
+
+def uptime_string_to_seconds(uptime_string):
+    """Converts uptime string seconds.
```
Comment (jeffkala) [inline netutils/time.py:33]:
```suggestion
    """Converts human readable uptime string to number of seconds.
```

**Review by jeffkala — CHANGES_REQUESTED** [review]

**Comment by qduk** [comment]
@itdependsnetworks  Want to give this another look over?

**Pull request merged**